src/classes/AFFL_MultiRecordType_TEST.cls

```diff
@@ -416,4 +416,49 @@ public with sharing class AFFL_MultiRecordType_TEST {
 		System.assertEquals(false, result.success);
 		System.assertEquals(Label.afflAccoutMappingError, result.errors[0].message);
 	}
+
+	@isTest
```

Comment:
I know we're not consistent in this class, but we should be adding ApexDoc-style comments to every new unit test so that it's clear what each unit test is testing.